src/jit/importer.cpp

```diff
@@ -7492,17 +7492,6 @@ GenTreePtr                Compiler::impFixupCallLongReturn(GenTreePtr     call,
     unsigned retRegCount = retTypeDesc->GetReturnRegCount();
     // must be a long returned in two registers
     assert(retRegCount == 2);
-
```

Comment:
Yes, I would like that change.  But I am equally fine if you don't want to club that change with this PR.